client/lib/application/src/application-list/application-list.hbs

```diff
@@ -3,6 +3,19 @@
     <a href="/application/create" class="btn btn-default">
     <i class="fa fa-plus"></i> Create Application</a>
 </div>
+<form>
```

Comment:
I’m not particularly impressed by the styling of the form…

Proposals:

* Give the form the class `form` for some default styles.
* Put `button` and `input` in an `div.input-group` to align them again on the same vertical level
* Put `label` (see below) and `.input-group` in `div.form-group` for some default styles

There’d be some additional adjustments to make, e.g. max-width of the view and removing left border of button in input-group.